tests/tests_e3/anod/spec_test.py

```diff
@@ -50,6 +153,35 @@ def build(self):
     assert len(ms.deps) == 0
 
 
+@pytest.mark.parametrize("arguments,expected", CHECK_DLL_CLOSURE_ARGUMENTS)
+def test_spec_check_dll_closure(arguments: tuple, expected: tuple) -> None:
+    """Create a simple spec with dependency to python and run dll closure."""
+    ldd_output, ignored = arguments
+    (errors,) = expected
+    test_spec: Anod = Anod("", kind="install")
+    test_spec.sandbox = SandBox(root_dir=os.getcwd())
+    if ldd_output is None:
+        # Use the current executable lib directory.
+        exe_path: Path = Path(sys.executable)
+        lib_path: Path = Path(
+            exe_path.parent.parent, "lib" if sys.platform != "win32" else ""
+        )
+        test_spec.check_shared_libraries_closure(
+            prefix=str(lib_path), ignored_libs=ignored, ldd_output=ldd_output
```

Comment:
here ldd_output is None, so it would be better to write None for clarity

this test will fail on Windows system without ldd, it will also fail on macOS, can had a @pytest.skipif(not ldd)

Reply:
Do you prefer that I use `skipif` or that I define an `ldd = require_tool("ldd")` in `conftest.py` ? I would vote for the second, but to me both are equal.

Reply:
require_tool is fine with me